22498 - lint issue with previous fix for lsblk.go

**Describe the bug**
error in lint and unit test from PR #1095 

make lint gets the following error:

```
INFO [runner] linters took 24.048004155s with stages: goanalysis_metalinter: 24.039978211s 
pkg/base/linuxutils/lsblk/lsblk.go:163:68: unnecessary leading newline (whitespace)
```

`make test` error:
```
=== RUN   TestLSBLK_SearchDrivePath_Success
--- FAIL: TestLSBLK_SearchDrivePath_Success (0.00s)
panic:
assert: mock: I don't know what to return because the method call was unexpected.
        Either do Mock.On("RunCmd").Return(...) first, or remove the RunCmd() call.
        This method was unexpected:
                RunCmd(string)
                0: "lsblk  --paths --json --bytes --fs --output NAME,TYPE,SIZE,ROTA,SERIAL,WWN,VENDOR,MODEL,REV,MOUNTPOINT,FSTYPE,PARTUUID"
        at: [executors.go:181 lsblk.go:133 lsblk.go:166 lsblk_test.go:92] [recovered]
        panic:
assert: mock: I don't know what to return because the method call was unexpected.
        Either do Mock.On("RunCmd").Return(...) first, or remove the RunCmd() call.
        This method was unexpected:
                RunCmd(string)
                0: "lsblk  --paths --json --bytes --fs --output NAME,TYPE,SIZE,ROTA,SERIAL,WWN,VENDOR,MODEL,REV,MOUNTPOINT,FSTYPE,PARTUUID"
        at: [executors.go:181 lsblk.go:133 lsblk.go:166 lsblk_test.go:92]
```
**Environment (please complete the following information):**
 - Hardware: [e.g. Dell PowerEdge]
 - OS: [e.g. Ubuntu, run `printf "$(uname -srm)\n$(cat /etc/os-release)\n"`]
 - Version: [e.g. 1.0.0]
Lint eError

**To Reproduce**
```
make lint 
```

**Expected behavior**
It should build

**Screenshots**
look up 

**Additional context**
Add any other context about the problem here.

--- a/pkg/base/linuxutils/lsblk/lsblk.go
+++ b/pkg/base/linuxutils/lsblk/lsblk.go
@@ -161,7 +161,13 @@ func (l *LSBLK) GetBlockDevices(device string) ([]BlockDevice, error) {
 // Receives an instance of drivecrd.Drive struct
 // Returns drive's path based on provided drivecrd.Drive or error if something went wrong
 func (l *LSBLK) SearchDrivePath(drive *api.Drive) (string, error) {
-	device := ""
+	// device path might be already set by hwmgr
+	device := drive.Path
+	if device != "" {
+		return device, nil
+	}
+
+	device = ""
 	// try to find it with lsblk
 	lsblkOut, err := l.GetBlockDevices("")
 	if err != nil {

--- a/pkg/base/linuxutils/lsblk/lsblk.go
+++ b/pkg/base/linuxutils/lsblk/lsblk.go
@@ -1,5 +1,5 @@
 /*
-Copyright © 2020 Dell Inc. or its subsidiaries. All Rights Reserved.
+Copyright © 2020-2024 Dell Inc. or its subsidiaries. All Rights Reserved.
 
 Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
@@ -161,7 +161,6 @@ func (l *LSBLK) GetBlockDevices(device string) ([]BlockDevice, error) {
 // Receives an instance of drivecrd.Drive struct
 // Returns drive's path based on provided drivecrd.Drive or error if something went wrong
 func (l *LSBLK) SearchDrivePath(drive *api.Drive) (string, error) {
-
 	device := ""
 	// try to find it with lsblk
 	lsblkOut, err := l.GetBlockDevices("")

--- a/pkg/base/linuxutils/lsblk/lsblk.go
+++ b/pkg/base/linuxutils/lsblk/lsblk.go
@@ -161,13 +161,7 @@ func (l *LSBLK) GetBlockDevices(device string) ([]BlockDevice, error) {
 // Receives an instance of drivecrd.Drive struct
 // Returns drive's path based on provided drivecrd.Drive or error if something went wrong
 func (l *LSBLK) SearchDrivePath(drive *api.Drive) (string, error) {
-	// device path might be already set by hwmgr
-	device := drive.Path
-	if device != "" {
-		return device, nil
-	}
-
-	device = ""
+	device := ""
 	// try to find it with lsblk
 	lsblkOut, err := l.GetBlockDevices("")
 	if err != nil {

--- a/pkg/base/linuxutils/lsblk/lsblk_test.go
+++ b/pkg/base/linuxutils/lsblk/lsblk_test.go
@@ -84,14 +84,6 @@ func TestLSBLK_SearchDrivePath_Success(t *testing.T) {
 	e := &mocks.GoMockExecutor{}
 	l := NewLSBLK(testLogger)
 	l.e = e
-	// path is in drive spec
-	dCR := testDriveCR
-	path := "/dev/sda"
-	dCR.Spec.Path = path
-
-	res, err := l.SearchDrivePath(&dCR.Spec)
-	assert.Nil(t, err)
-	assert.Equal(t, path, res)
 
 	// got from lsblk output
 	e.On("RunCmd", allDevicesCmd).Return(mocks.LsblkTwoDevicesStr, "", nil)
@@ -100,7 +92,7 @@ func TestLSBLK_SearchDrivePath_Success(t *testing.T) {
 	d2CR := testDriveCR
 	d2CR.Spec.SerialNumber = sn
 
-	res, err = l.SearchDrivePath(&d2CR.Spec)
+	res, err := l.SearchDrivePath(&d2CR.Spec)
 	assert.Nil(t, err)
 	assert.Equal(t, expectedDevice, res)
 }

